src/JasperBus/IServiceBus.cs

```diff
@@ -21,15 +21,15 @@ public interface IServiceBus
         /// <returns></returns>
         Task<TResponse> Request<TResponse>(object request, RequestOptions options = null);
 
-        void Send<T>(T message);
+        Task Send<T>(T message);
```

Comment:
I'm totally on board with making the sending pipeline underneath this being async all the way down, but I'm not sure we want to force this on the IServiceBus interface